### PR TITLE
DEP Dependencies for CMS 6

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/.github/issues/191

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -7,11 +7,11 @@
     "require": {
         "php": "^8.1",
         "silverstripe/recipe-plugin": "2.x-dev",
-        "silverstripe/recipe-cms": "5.x-dev",
-        "silverstripe/documentconverter": "3.x-dev",
-        "silverstripe/iframe": "3.x-dev",
-        "silverstripe/tagfield": "3.x-dev",
-        "silverstripe/taxonomy": "3.x-dev"
+        "silverstripe/recipe-cms": "6.x-dev",
+        "silverstripe/documentconverter": "4.x-dev",
```
Comment (ChloeSartorelli) [inline composer.json:11]: There's no way we're making document converter CMS 6 compatible, so remove that.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json:11]: We'll create a CMS 6 branch for now and look at support at a later date

Comment (ChloeSartorelli) [reply composer.json:11]: That's ridiculous but sure.

**Review by GuySartorelli — APPROVED** [review]
LGTM. Failures are expected at this stage so ignoring CI.

**Pull request merged**